dictionary/data_world_bank.csv

```diff
@@ -46,12 +46,12 @@ Comoros,COM
 "Congo, Dem. Rep.",COD
 "Congo, Rep.",COG
 Costa Rica,CRI
-Côte d'Ivoire,CIV
 Croatia,HRV
 Cuba,CUB
 Curaçao,CUW
 Cyprus,CYP
 Czech Republic,CZE
+Côte d’Ivoire,CIV
```

Comment:
Do we want curly ' ?

Reply:
I think it is probably not proper, right? But that's what it is in the original data. Should I make the scraper "fix" it?

Reply:
Yeah, I think so. My gut feeling is that converting to nice curly should be the typesetter's job, not a data-level thing.

Reply:
🤔 
``` r
packageVersion("countrycode")
#> [1] '1.5.0'
countrycode::countrycode("CIV", "iso3c", "country.name")
#> [1] "Côte d’Ivoire"
```

Reply:
It's hard to see the difference with GitHub's formatting, but both ASCII single quote and UTF curly quote work, and UTF curly quote is what's currently in `countrycode::codelist`, so I'm assuming leaving it as curly quote is ok, maybe even ideal.
``` r
library(countrycode)
packageVersion("countrycode")
#> [1] '1.6.0'

as.hexmode(utf8ToInt("'"))
#> [1] "27"
as.hexmode(utf8ToInt("’"))
#> [1] "2019"

countrycode("Côte d\U27Ivoire", "country.name", "country.name")
#> [1] "Côte d’Ivoire"
countrycode("Côte d\U2019Ivoire", "country.name", "country.name")
#> [1] "Côte d’Ivoire"

stringi::stri_escape_unicode(countrycode("CI", "iso2c", "country.name"))
#> [1] "C\\u00f4te d\\u2019Ivoire"
```